test/platform/test-platform-identify.ts

```diff
@@ -0,0 +1,131 @@
+/// <reference types="@google/local-home-sdk" />
+import test from 'ava';
+import {
+  MockLocalHomePlatform,
+  ERROR_UNDEFINED_IDENTIFYHANDLER,
+  ERROR_UNDEFINED_APP,
+  ERROR_LISTEN_NOT_CALLED,
+  ERROR_UNDEFINED_VERIFICATIONID,
+} from '../../src/platform/mock-local-home-platform';
+import {AppStub} from '../../src/platform/smart-home-app';
+import {injectSmarthomeStubs} from '../../src';
+
+const DISCOVERY_BUFFER = Buffer.from('discovery buffer 123');
+const APP_VERSION = '0.0.1';
+const DEVICE_ID = 'device-id-123';
+
+test.before(t => {
+  injectSmarthomeStubs();
+});
+
+test('trigger-identify-with-undefined-app-throws', async t => {
+  const mockLocalHomePlatform = MockLocalHomePlatform.getInstance(true);
+  t.is(
+    (
+      await t.throwsAsync<Error>(
+        mockLocalHomePlatform.triggerIdentify(DISCOVERY_BUFFER)
+      )
+    ).message,
+    ERROR_UNDEFINED_APP
+  );
+});
+
+test('trigger-identify-without-listen-throws', async t => {
+  const mockLocalHomePlatform = MockLocalHomePlatform.getInstance(true);
+  const app: AppStub = new AppStub(APP_VERSION);
+  t.is(
+    (
+      await t.throwsAsync<Error>(
+        mockLocalHomePlatform.triggerIdentify(DISCOVERY_BUFFER)
```

Comment:
Can you explain why this may be an error?

Reply:
Added comments explaining this.  `listen()` indicates all handlers have been set by setting a flag.  `MockLocalHomePlatform` returns this error when the flag has not been set prior to calling the handler.